src/openforms/prefill/contrib/objects_api/api/views.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any
+
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from rest_framework import authentication, permissions, views
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from openforms.contrib.objects_api.api.serializers import ObjectsAPIGroupInputSerializer
+from openforms.contrib.objects_api.api.views import (
+    ObjecttypesListView,
+    ObjecttypeVersionsListView,
+)
+from openforms.contrib.objects_api.clients.builder import get_objecttypes_client
+from openforms.contrib.objects_api.json_schema import (
+    InvalidReference,
+    iter_json_schema_paths,
+)
+
+from .serializers import PrefillTargetPathsSerializer
+
+OBJECTS_API_GROUP_QUERY_PARAMETER = OpenApiParameter(
+    name="objects_api_group",
+    type=OpenApiTypes.STR,
+    location=OpenApiParameter.QUERY,
+    description=_("Which Objects API group to use."),
+)
+
+
+@extend_schema(
+    summary=_("List available objecttypes for Objects API"),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PrefillObjectsAPIObjecttypeListView(ObjecttypesListView):
+    """
+    List the available prefill objecttypes for Objects API plugin.
+    """
+
+
+@extend_schema(
+    summary=_("List available objecttype versions for Objects API"),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PrefillObjectsAPIObjecttypeVersionListView(ObjecttypeVersionsListView):
+    """
+    List the available prefill objecttype versions for Objects API plugin.
+    """
+
+
+@extend_schema(
+    summary=_("List available attributes for Objects API"),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PrefillObjectsAPIAttributesListView(views.APIView):
```

Comment:
```suggestion
class ObjecttypePropertiesListView(views.APIView):
```
we're already in the `prefill.contrib.objects_api` package, so we can omit that information and instead be more specific: we're retrieving a list of available JSON properties defined on a particular objecttype